docs/data/toolpad/core/components/sign-in-page/sign-in-page.md

```diff
@@ -0,0 +1,109 @@
+---
+productId: toolpad-core
+title: Sign In Page
+components: SignInPage, Account, NotificationsProvider
+---
+
+# Sign In Page
+
+<p class="description">A component that renders a functional authentication page for your application.</p>
+
+The `SignInPage` component is a quick way to generate a ready-to-use authentication page with multiple OAuth providers, or a credentials form.
+
+## OAuth
+
+The `SignInPage` component can be set up with an OAuth provider by passing in a list of providers in the `providers` prop, along with a `signIn` function that accepts the `provider` as a parameter.
```

Comment:
_Can be done in follow-up_

We should also explain which providers are supported